app/packs/review/_components/UnreviewedPack.tsx

```diff
@@ -123,8 +121,8 @@ export default function UnreviewedPack({
 					>
 						<AlertDialogTrigger asChild>
 							<Button
-								variant="default"
-								className="bg-emerald-500 hover:bg-emerald-600 text-white"
+								variant="success"
+								className="bg-emerald-400"
```

Comment:
Don't set a custom background, theme for the button should all be located in its variant in the button component
```suggestion
```